pe-8655 addendum: updating multi-nic routing info to edge and launchpad for VMs (#10740)
spectrocloud/librarium: docs/docs-content/vm-management/launchpad-for-vms/vmo-networking.md

## Management and Cluster Traffic

Launchpad for VMs hosts can use separate network interfaces for management traffic and Kubernetes cluster traffic.
Management traffic includes Local UI access, communication between hosts, and content synchronization.

You can configure the management interface in the Edge Installer `user-data` file. A selection made in the TUI overrides
the value from `user-data`. After initial setup, a selection made in Local UI overrides the value from the TUI. If no
management interface is selected, the host uses the network interface associated with the default route.

Kubernetes cluster traffic uses the interface selected during cluster creation. This includes node IP selection,
Kubernetes control plane traffic, etcd traffic, and traffic for the cluster virtual IP address (VIP). For Launchpad for
VMs, this is the interface selected in the **Cilium and MetalLB interface** field.

:::warning

Selecting a management interface or a cluster traffic interface does not change how the host routes network traffic. If
multiple adapters use the same subnet and each adapter has a default route, traffic may leave through a different

> adapter than expected. This can cause control plane nodes to fail joining the cluster.

Reviewer: ```suggestion
adapter than expected. This may prevent control plane nodes from successfully joining the cluster.
```

Reviewer: Maybe?
